README.md

```diff
@@ -67,9 +67,21 @@ To complete the steps in this tutorial, you need to install Node.js and the Loop
 
    If you don't have a development environment, we'd recommend using [Visual Studio Code](https://code.visualstudio.com/). The LoopBack application generator CLI allows you to generate the VSCode setting files.
 
+## How to use this repo and its branches
+
+This repository contains different branches for various stages of this workshop.
+The master branch only contains the instructions (README.md) for this workshop.
+The `workshop-part1-completed` branch contains the instructions (README.md) for this workshop as well as all the artifacts completed in Part 1 on the workshop. The same is true for the remaining `workshop-part{2,3,4}-completed` branches.
+In the workshop, you will `git clone` this repository and start adding code with the master branch.
+Every time you finish a section, you can run `git checkout <the_next_step>` to begin the next section.
+For example, when following the first section, you will be on branch `master`. After the app is created, you can run `git checkout workshop-part1-complete` which contains our default implementation for section 1 to begin section 2.
```

Comment:
If users are not familiar with `git`, I think it's better to have 
```
git add .
git stash // or git commit, I think stash is easier
git checkout.....
```
^ I have these 3 steps in slides so maybe we don't need them in the instruction.

Reply:
I will make it consistent with slides.